lib/bpk-component-emoji/stories.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/accessible-emoji */
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { Platform } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import CenterDecorator from '../../storybook/CenterDecorator';
+import BpkText from '../bpk-component-text';
+
+import BpkEmoji from './index';
+
+storiesOf('bpk-component-emoji', module)
+  .addDecorator(CenterDecorator)
+  .add('docs:default', () => (
+    <BpkText style={{ fontFamily: Platform.OS === 'ios' ? 'Didot' : 'System' }}>
```

Comment:
Should we try and be using Relative here for the font family so that we can see how it will look with our font as that is what the consumers would see when using it?

Reply:
Yeah but currently our example app doesn't support it. When it does, our example can just be `<BpkText> Yay ... <BpkEmoji>🎉</BpkEmoji></BpkText>`

Reply:
I thought our example app does support Relative already?

https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md#using-relative-font